client/my-sites/domains/domain-management/domain-mapping-instructions/index.jsx

```diff
@@ -0,0 +1,143 @@
+/**
```

Comment:
As discusses on call, maybe let's put it in `my-sites/domains/components/mapping-instructions`, since this will be shared between domain management and checkout :bow: